### PR TITLE
Update README.rst with notes about bind mounts

### PR DESCRIPTION
**Pull request by rediris**
### Bind Mounts
- adds a note detailing the necessity of the bind-mount folder name matching the spelling of the repo name.
- Please see the discussion at [Tutor [Palm]: How to bind-mount a local MFE folder](https://discuss.openedx.org/t/tutor-palm-how-to-bind-mount-a-local-mfe-folder/10734/4)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -301,6 +301,25 @@ This works for custom MFEs, as well. For example, if you added your own MFE name
 
 Similarly, in production, the "mfe" Docker image will be rebuilt automatically during ``tutor local launch``.
 
+Note: the name of the bind-mount folder needs to match the name of the repository exactly word-for-word. If you've forked an MFE repository with a custom name, be sure to change the name back to ensure the bind-mount works properly.
+
+Using shared components for local development
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Developers may want to use local repositories of common components such as `frontend-component-footer <https://github.com/openedx/frontend-component-footer>`_ and `frontend-component-header <https://github.com/openedx/frontend-component-header>`_. In the following example, let's assume one has cloned the example MFE and the shared components into a directory called ``workspace``. In the root of the example MFE, create a file called ``module.config.js`` and use the following to ensure your example MFE picks up the locally hosted components during build time:
```
Comment (regisb) [inline README.rst:309]: My understanding is that this paragraph only partially explains how to work on shared components. How is the "frontend-component-header" bind-mounted in the container at runtime? And at build-time? How did you get it to work _exactly_?

I think that your explanations would be valuable to other users, but we need to provide comprehensive instructions.

Comment (rediris) [reply README.rst:309]: Hi @regisb, you're right - I'm having trouble reproducing the steps even though I had gotten it to work at some point. I'll update the PR to omit the shared components bit and I'll add another PR in the future when I figure out how to get it working again.

**Review by regisb — APPROVED** [review]
LGTM! Passing on to @arbrandes :)

File: README.rst [hunk 2]
```diff
@@ -329,6 +329,8 @@ This works for custom MFEs, as well. For example, if you added your own MFE name
 
 Similarly, in production, the "mfe" Docker image will be rebuilt automatically during ``tutor local launch``.
 
+Note: the name of the bind-mount folder needs to match the name of the repository exactly word-for-word. If you've forked an MFE repository with a custom name, be sure to change the name back to ensure the bind-mount works properly.
```
Comment (arbrandes) [inline README.rst:332]:
```suggestion
Note: the name of the bind-mount folder needs to match the name of the repository word-for-word. If you've forked an MFE repository with a custom name, be sure to change the name back to ensure the bind-mount works properly.
```

Nit-pick: "exactly word-for-word" feels a bit redundant.

**Review by arbrandes — CHANGES_REQUESTED** [review]
Works for me, though I do have one nit minor with the wording.

**Comment by rediris** [comment]
@arbrandes thanks for the suggestion, and this is all set for your review.

**Pull request merged**